templates/client-auth-n.cfg.xml.erb

```diff
@@ -3,7 +3,7 @@
 <client-auth xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/client-auth/v1.0">
 <%- if ! @delegating.nil? -%>
   <%- if @delegating %>
-    <delegating/>
+    <delegating<%- if ! @delegating_quality.nil? -%> quality="<%= @delegating_quality %>"<%- end -%>/>
```

Comment:
Is that the case @mwhahaha ?  The spec test verifies that the space exists @ https://github.com/usnavi/puppet-repose/blob/CF-903_adding_quality/spec/defines/filter/client_auth_n_spec.rb#L207

Or am i misunderstanding something?